test/sql/catalog/sequence/test_sequence.test

```diff
@@ -489,3 +489,7 @@ select nextval(1 + 1);
 
 statement error
 select currval(true);
+
+# max value specified more than once
+statement error
+CREATE SEQUENCE wrongseq NO MAXVALUE MAXVALUE 2;
```

Comment:
Can we add tests for all of the (potentially) duplicated sequence elements?

Reply:
Added them